src/components/Categories.js

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
-import ProductCategoriesData from '../mocks/en-us/product-categories.json';
+import React, { useContext } from 'react';
 import { Category, CategoriesContainer } from './Categories.styles';
+import { useApi } from '../utils/hooks/useApi';
+import ThemeContext from '../context/ThemeContext';
+
+export default function Categories() {
+  const response = useApi('category', 10, 1);
+
```

Comment:
Qué tal si aqui mismo creamos una constante que traiga lo que queremos, así evitaremos en el código de abajo tener que escribir `reponse?.data?.results` 2 veces.
```suggestion
  const response = useApi('category', 10, 1);
  const categories = response?.data?.results;
```

Reply:
Si, voy a aplicar esto con todos los response! gracias